### PR TITLE
Create ui-c8y-1018-503-96-device-data-widget-with-long-software-list-fix-6475-graft-release.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-3669](https://cumulocity.atlassian.net/browse/DM-3669)
Original PR: [6500](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6500)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1018-503-96-device-data-widget-with-long-software-list-fix-6475-graft-release.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Device data widget with long software list fix (#6475) [GRAFT][release/y2024] (#6500)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1018-503-96-device-data-widget-with-long-software-list-fix-6475-graft-release.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fixed an issue with the device data widget when displaying a long software list
```

File: content/change-logs/device-management/ui-c8y-1018-503-96-device-data-widget-with-long-software-list-fix-6475-graft-release.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3669
+version: 1018.503.96
+---
+Device data widget with long software list fix (#6475) [GRAFT][release/y2024] (#6500)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1018-503-96-device-data-widget-with-long-software-list-fix-6475-graft-release.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the device data widget, the software list was not displayed correctly if it contained a large number of entries, making it difficult for users to view and manage the installed software on their devices. This issue has now been resolved. The device data widget now properly displays long software lists, ensuring that users have full visibility of all installed software, regardless of the number of entries. This improvement enhances the usability and functionality of the device management capabilities.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/device-management/ui-c8y-1018-503-96-device-data-widget-with-long-software-list-fix-6475-graft-release.md (hunk 1 above)
Comment (dafkk) [inline content/change-logs/device-management/ui-c8y-1018-503-96-device-data-widget-with-long-software-list-fix-6475-graft-release.md:3]:
```suggestion
title: The device data widget with a long software list is fixed by transforming the software list into a paginated one.
```

File: content/change-logs/device-management/ui-c8y-1018-503-96-device-data-widget-with-long-software-list-fix-6475-graft-release.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (dafkk) [inline content/change-logs/device-management/ui-c8y-1018-503-96-device-data-widget-with-long-software-list-fix-6475-graft-release.md:17]:
```suggestion
In the device data widget, the software list was not displayed correctly if it contained a large number of entries. This issue has now been resolved by restricting the initial software list by only 10 items and an option to load more if necessary.
```

File: content/change-logs/device-management/ui-c8y-1018-503-96-device-data-widget-with-long-software-list-fix-6475-graft-release.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3669
+version: 1018.503.96
+---
+In the device data widget, the software list was not displayed correctly if it contained a large number of entries. This issue has now been resolved by restricting the initial software list by only 10 items and an option to load more if necessary.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1018-503-96-device-data-widget-with-long-software-list-fix-6475-graft-release.md:17]:
```suggestion
In the "Device data" widget, the software list was not displayed correctly if it contained a large number of entries. This issue has now been resolved by paginating the software list, that is, limiting the initial list to 10 items and an option to load more if needed.
```

File: content/change-logs/device-management/ui-c8y-1018-503-96-device-data-widget-with-long-software-list-fix-6475-graft-release.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: The device data widget with a long software list is fixed by transforming the software list into a paginated one.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1018-503-96-device-data-widget-with-long-software-list-fix-6475-graft-release.md:3]:
```suggestion
title: Software list in Device data widget now paginated to show all entries properly
```

**Review by bozhidardedov — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**